apps/mochi-web/pages/explore.tsx

```diff
@@ -11,7 +12,11 @@ export default function ExplorePage() {
 
   return (
     <Layout>
-      <SEO title="Explore" />
+      <SEO
+        title="Explore Mochi transactions"
+        description="Let's see how Mochi citizens are doing!"
+        image={`${HOME_URL}/assets/explore/explore.thumbnail.png`}
```

Comment:
Not sure why we need to have `HOME_URL`. I think it should still work without the variable.